docs/01_key-concepts/03_cpu.md

```diff
@@ -1,6 +1,10 @@
 ---
-content_title: CPU as resource
-link_text: CPU as resource
+content_title: CPU as system resource
+link_text: CPU as system resource
 ---
 
-CPU is processing power, the amount of CPU an account has is measured in microseconds, it is referred to as `cpu bandwidth` on the cleos get account command output and represents the amount of processing time an account has at its disposal when pushing actions to a contract.
+The system resource CPU provides processing power to blockchain accounts. The amount of CPU an account has is measured in microseconds and it is referred to as `cpu bandwidth` on the `cleos get account` command output. The `cpu bandwidth` represents the amount of processing time an account has at its disposal when pushing actions to a contract. When a transaction is executed by the blockchain it consumes CPU and NET, therefore sufficient CPU must be staked in order for transactions to complete.
```

Comment:
suggestion:
use italics if you want to emphasize a non-proper name or term:
\`cpu bandwidth\` --> \*cpu bandwidth\*